test/sender_controller_test.dart

```diff
@@ -42,16 +43,34 @@ main() async {
  test(
    "Test for add/delete file",
      () {
+        final file = _TestingFile(fileName: "fuck you", fileSize: 100);
         expect(controller.getFiles(), []);
-        controller.onAddNewFile("fuck you");
-        expect(controller.getFiles(), ["fuck you"]);
+        controller.onAddNewFile(file);
+        expect(controller.getFiles(), [file]);
         controller.onDeleteFile(0);
         expect(controller.getFiles(), []);
      }
  );
 }
 
-class TestingModel implements ISenderModel<String> {
+class _TestingFile extends AbstractFile {
+  final String fileName;
+  final int fileSize;
+
+  _TestingFile({this.fileName, this.fileSize});
+
+  @override
+  String getFileName() {
+    return this.fileName;
+  }
+
+  @override
+  int getFileSize() {
+    return this.fileSize;
+  }
```

Comment:
アロー定期 (後からなんか付け足す予定とかだったら申し訳ない)

```suggestion
  @override
  String getFileName() => this.fileName;

  @override
  int getFileSize() => this.fileSize;
```

Reply:
ああここは確かにそうだ